pkg/trustyapi/trustyapi.go

```diff
@@ -161,6 +160,22 @@ func ProcessDependency(dep string, ecosystem string, scoreThreshold float64) (st
 
 	reportBuilder.WriteString(fmt.Sprintf("### 📉 Trusty Score: `%.2f`\n", result.Summary.Score))
 
+	// write provenance information
+	if result.Provenance.Description.Provenance.Issuer != "" {
+		reportBuilder.WriteString("### :key: Proof of origin (Provenance):\n")
+		reportBuilder.WriteString("Built and signed with sigstore using GitHub Actions.\n")
+		reportBuilder.WriteString(fmt.Sprintf("· Source repo: `%s`\n", result.Provenance.Description.Provenance.SourceRepo))
+		reportBuilder.WriteString(fmt.Sprintf("· Github Action Workflow: `%s`\n", result.Provenance.Description.Provenance.Workflow))
+		reportBuilder.WriteString(fmt.Sprintf("· Issuer: `%s`\n", result.Provenance.Description.Provenance.Issuer))
+		reportBuilder.WriteString(fmt.Sprintf("· Rekor Public Ledger: `%s`\n", result.Provenance.Description.Provenance.Transparency))
+	} else {
+		// need to write regular provenance info
+		reportBuilder.WriteString("### :key: Proof of origin (Provenance):\n")
+		reportBuilder.WriteString(fmt.Sprintf("# versions: %.0f\n", result.Provenance.Description.Hp.Versions))
+		reportBuilder.WriteString(fmt.Sprintf("# tags: %.0f\n", result.Provenance.Description.Hp.Tags))
+		reportBuilder.WriteString(fmt.Sprintf("# matched: %.0f\n", result.Provenance.Description.Hp.Common))
+	}
+
```

Comment:
We should add some context what provenance means, could we maybe include the following link o lines 171 and 177 

https://docs.stacklok.com/trusty/understand/provenance

You can use the text 'Learn more about source of origin provenance'